src/components/detail/Detail.vue

```diff
@@ -301,6 +180,10 @@ export default {
             const h = this.highlightMetric
             router.push('/' + h.area + '/' + h.name)
             $('.ui.metrics.modal').modal('hide')
+        },
+
+        setTimeRange (newRange) {
```

Comment:
I think we should make a time range widget that takes v-model="metricData.range" or something like that